packages/components/src/components/lifecycleValidation.spec.ts

```diff
@@ -136,6 +148,20 @@ it.each<TagName>(tagNamesPublicWithoutProps)('should not call validateProps() fo
   expect(spy).not.toBeCalled();
 });
 
+it.each<TagName>(tagNamesWithPropsOfTypeObject)(
+  'should call isDeepEqual() with correct parameters for %s',
```

Comment:
To match other test names:
```suggestion
  'should call isDeepEqual() with correct parameters via componentShouldUpdate for %s',
```